utils/testing.ts

```diff
@@ -60,6 +60,10 @@ export async function compareFormData(formData: Record<string, unknown>, entries
  */
 export function extractNonSkylinkPath(url: string, skylink: string): string {
   const parsed = parse(url, {});
-  const path = parsed.pathname.replace(skylink, ""); // Remove skylink to get the path.
-  return trimForwardSlash(path);
+  let path = parsed.pathname.replace(skylink, ""); // Remove skylink to get the path.
+  path = trimForwardSlash(path);
+  if (path != "") {
```

Comment:
use `!==` instead

in any case, in what case this condition is needed? If `path` is not empty then the pathname was separated from skylink by a `/` and `path` will be prepended by `/`.

Reply:
This is just ensuring there is only one leading slash. Added some comments.